MISSING_SEQUENCE_DETECTION.md

```diff
@@ -0,0 +1,117 @@
+# Missing interchange sequence detection
+
+In rare occasion of an in-code exception there is a chance, 
+that a sequence number will be generated and registered in NHAIS database as used
+but due to the error no message will be sent with that number.
+
+Such situation will produce gaps in sequence numbers sent to HA
+
+This document describes how to produce a report on missing sequence numbers.
+
```

Comment:
Can I suggest changing this wording to: 
It is possible, however unlikely, that the adaptor may generate a sequence number before encountering an exception and aborting the transaction.  In such cases, the sequence number then becomes redundant and can cause the sequencing to break with gaps being sent onto the HA and stored within the adaptors state database.  This could happen for both Outbound and Inbound transactions.